source/index.txt

```diff
@@ -28,7 +28,10 @@ Introduction
 Welcome to the documentation site for the official {+driver-long+}.
 You can add the driver to your application to work with MongoDB
 in JavaScript. Download it using `npm <https://www.npmjs.com/>`__
-or set up a runnable project by following our Quick Start guide.
+or set up a runnable project by following our Quick Start guide. For more
```

Comment:
Suggestion:

"... work with MongoDB in JavaScript or TypeScript."

Omit the next sentence about "Download" or "Set up a Runnable project" since the next sentence provides that information and the npm link is misleading.